src/Juvix/Compiler/Core/Extra/Recursors/Map/Named.hs

```diff
@@ -0,0 +1,95 @@
+module Juvix.Compiler.Core.Extra.Recursors.Map.Named where
+
+import Juvix.Compiler.Core.Extra.Recursors.Base
+import Juvix.Compiler.Core.Extra.Recursors.Map
+import Juvix.Compiler.Core.Extra.Recursors.Parameters
+
+dmapLRM :: Monad m => (BinderList Info -> Node -> m Recur) -> Node -> m Node
```

Comment:
I see you changed *B to *L and changed the order, so now M is always at the end if present. That's fine, but the suffixes in Fold/Named.hs should be made consistent with this convention.